service/src/test/java/org/mskcc/cmo/metadb/service/MetadbServiceTest.java

```diff
@@ -207,5 +213,26 @@ public void testSampleHistoryListIsAscendingByImportDate() throws Exception {
         List<SampleMetadata> sampleMetadataHistory = sampleService.getSampleMetadataHistoryByIgoId(igoId);
         Assertions.assertThat(sampleMetadataHistory).isSorted();
     }
+    
+    @Test
+    public void testFindPatientByPatientAlias() {
+        Assertions.assertThat(patientRepository.findPatientByPatientAlias("C-MP789JR")).isNotNull();
+    }
+    
+    @Test
+    public void testFindPatientByPatientAliasWithExpectedFailure() {
+        MetaDbPatient patient = new MetaDbPatient();
+        PatientAlias patientAlias = new PatientAlias();
+        patientAlias.setPatientId("C-MP789JR");
+        patient.addPatientAlias(patientAlias);
+        patientRepository.save(patient);
+        
+        try {
+            MetaDbPatient retrievedPatient = patientRepository.findPatientByPatientAlias("C-MP789JR");
+            Assert.fail();
```

Comment:
you can remove this try/catch black and instead add an annotation to this test method that indicates it's supposed to throw a specific exception (not just a generic Exception)

the annotation is something like `@ExpectedException` or something like that